### DISCUSSION TITLE
Sharepoint syncing , specify a sharepoint folder

### DISCUSSION THREAD

**Discussion by Stephan35** [abraunegg/onedrive#1842] [Q&A] [answered]
Hello,
I would like to syncing a remote folder on sharepoint, not in root,
something like /General/folder_master/this_folder
is there any way to do it ?
Thanks

**Comment by abraunegg** [comment, marked as answer]
@Stephan35 
This is possible, please read the documentation on how to do this

**Reply by Stephan35** [reply]
Hello @abraunegg ,
You mean [this part](https://github.com/abraunegg/onedrive/blob/master/docs/advanced-usage.md#upload-data-from-the-local-onedrive-folder-to-a-specific-location-on-onedrive) of documentation ? (Upload data from the local ~/OneDrive folder to a specific location on OneDrive)

What i exatly wish to do : 
on my local drive , mounted NFS drive on /mnt/Share1/AFF/Work1
On my sharepoint , /General/folder_master/Work1
I would like to syncing Work1 from sharepoint to Work1 on my NFS drive.

sync_dir to "/mnt/Share1/AFF/Work1" , but put all directory from sharepoint in Work1 when syncing.
/mnt/Share1/AFF/Work1/General/folder_master/Work1/xxxxx is not what i look for.

**Reply by abraunegg** [reply]
You need to review this: https://github.com/abraunegg/onedrive/blob/master/docs/advanced-usage.md#upload-data-from-the-local-onedrive-folder-to-a-specific-location-on-onedrive

There are specific steps required

**Reply by abraunegg** [reply]
> sync_dir to "/mnt/Share1/AFF/Work1" , but put all directory from sharepoint in Work1 when syncing.
> /mnt/Share1/AFF/Work1/General/folder_master/Work1/xxxxx is not what i look for.

Because you are not following the instructions correctly
